siman/calc_manage.py

```diff
@@ -1229,7 +1229,7 @@ def add_loop_finalize(u_scale_flag, fitted_v100_id):
             #by add_calculation; for u-ramping names are different
             cl = calc[it, setlist[0], 1]
 
-            calc[fitted_v100_id].path["output"] = cl.path["output"].replace('/1.', '/100.')
+            calc[fitted_v100_id].path["output"] = cl.path["output"].replace('./1.', './100.')
```

Comment:
lets discus this change
calc[fitted_v100_id].path["output"] = cl.path["output"].replace('./1.', './100.')